receipt_dynamo/tests/integration/test__export_and_import.py

```
@@ -354,3 +356,153 @@ def test_export_image_does_not_exist(dynamodb_table, export_dir):
 def test_import_image_does_not_exist(dynamodb_table, export_dir):
     with pytest.raises(FileNotFoundError, match="JSON file not found"):
         import_image(dynamodb_table, f"{export_dir}/does_not_exist.json")
+
+
+@pytest.mark.integration
+def test_delete_image_data(dynamodb_table, export_dir):
```

Ruff (0.15.1)

[warning] Unused function argument: `export_dir`

(ARG001)